rs/config/src/subnet_config.rs

```diff
@@ -53,15 +52,18 @@ const INSTRUCTION_OVERHEAD_PER_CANISTER: NumInstructions = NumInstructions::new(
 const INSTRUCTION_OVERHEAD_PER_CANISTER_FOR_FINALIZATION: NumInstructions =
     NumInstructions::new(12_000);
 
-// If messages are short, then we expect about 2B=(7B - 5B) instructions to run
+// If messages are short, then we expect about 2B=(4B - 2B) instructions to run
```

Comment:
This is true for application subnets but not necessarily true for system subnets when install_code messages might be processed. Not sure if it's worth mentioning an exception but it could lead to some confusion if someone is looking at a system subnet.

Reply:
That's true, many constants here (i.e., not only this particular one) are only relevant for application subnets while system subnets have their own limits hard-coded directly in `fn system_subnets`. I'm not sure though if it's worthwhile to clean this up now (in this PR).

Reply:
Perhaps we simply mention that the comment refers mainly to application subnets and leave it at that for now.